### ISSUE TITLE
Model params and flops

### ISSUE THREAD

**Issue by taofuyu** [Deci-AI/super-gradients#1112]
### 📚 Documentation Improvement

Benchmarks only shows inference time on gpu, but the parameters and flops of model really matters.

**Comment by harpreetsahota204** [comment]
Agreed. 

In the meantime, if you are interested in this information, you can use[torchinfo](https://github.com/TylerYep/torchinfo) and get a model summary showing the number of parameters and flops.

**Comment by BloodAxe** [comment]
I respectfully disagree. Number of parameters and FLOPs actually matters much less than inference time since latency is the absolute measure of how fast model is. And FLOPs is just a proxy for this and number of parameters is even weaker proxy.

As @harpreetsahota204 pointed out you can use torchinfo, since we do not provide mechanism to compute the FLOPS. As for parameters count you can compute it with one line of code: 
```
    model = models.get(Models.YOLO_NAS_S, pretrained_weights="coco")
    trainable_parameters = sum([p.numel() for p in model.parameters() if p.requires_grad])
```
